src/sourcemaps/sourcemaps-wizard.ts

```diff
@@ -139,6 +144,11 @@ async function askForUsedBundlerTool(): Promise<SupportedBundlersTools> {
           value: 'vite',
           hint: 'Configure source maps upload using Vite',
         },
+        {
+          label: 'tsc',
```

Comment:
wdyt about

```suggestion
          label: 'Typescript (tsc)',
```

?

Reply:
I intentionally chose tsc to avoid ambiguity with people using e.g. typescript + webpack